integration/spark/src/main/scala/org/apache/spark/sql/execution/command/partition/CarbonAlterTableDropHivePartitionCommand.scala

```diff
@@ -105,6 +109,28 @@ case class CarbonAlterTableDropHivePartitionCommand(
             ifExists,
             purge,
             retainData).run(sparkSession)
+
+          // move  the partition files to trash folder which are dropped
+          val droppedPartitionNames = partitions.map { partition =>
+            partition.spec.map { specs => specs._1 + "=" + specs._2}
+          }
+          val droppedPartitionName = droppedPartitionNames
+            .head
+            .headOption
+            .get
+          TrashUtil.copyFilesToTrash(carbonPartitionsTobeDropped,
+            TrashUtil.getCompleteTrashFolderPath(
```

Comment:
`getCompleteTrashFolderPath` is used to create Path for Non-partition table(appends Segment Prefix), I think, for partition table, we should create Path with Partition folder name

Reply:
Added separate method for creating trash folder name with given partition folder names